### ISSUE TITLE
msp430-binutils forces linking

### ISSUE THREAD

**Issue by tpetry** [sampsyo/homebrew-mspgcc#8]
for building ``msp430-gcc`` you are forced to link ``msp430-binutils`` but this is a problem if you already have ``binutils`` installed.
It would be much better if ``msp430-binutils`` would be ``keg_only``.

**Comment by tpetry** [comment]
``keg_only`` would be stupid, because the bin-executables would not be found. But no executable of ``msp430-binutils`` conflicts with ``binutils`` so it would be nice if it could be linked directly. At the moment you have to use force the linking (which would overwrite duplicate files).

**Comment by sampsyo** [comment]
Yes, forcing the linking is unfortunate! I couldn't find a way around it, though, to install both packages under the same prefix. If someone better with the Homebrew API is watching, I'd love to remove that step.
